src/Component/Detail/DetailTodoList.js

```diff
@@ -0,0 +1,217 @@
+import CONSTANT from '../../Constants/Constans.js';
+import Component from '../../core/Component.js';
+import { delegate, on } from '../../util/helpers.js';
+import DetailButton from './DetailButton.js';
+
+export default class DetailTodoList extends Component {
+  constructor($element, props) {
+    super($element, props);
+
+    this.members = this.props.members;
+    this.addTeamTodoItem = this.props.addTeamTodoItem;
+    this.changeTeamTodoItemPriority = this.props.changeTeamTodoItemPriority;
+    this.toggleTeamTodoItem = this.props.toggleTeamTodoItem;
+    this.deleteTeamTodoItem = this.props.deleteTeamTodoItem;
+    this.editTeamTodoItemContents = this.props.editTeamTodoItemContents;
+    this.deleteTeamTodoItemAll = this.props.deleteTeamTodoItemAll;
+    this.changeFilter = this.props.changeFilter;
+    this.addTeamMember = this.props.addTeamMember;
```

Comment:
```suggestion
    const { member, 
        addTeamTodoItem, 
        changeTeamTodoItemPriority,  
        toggleTeamTodoItem,  
        deleteTeamTodoItem,
        editTeamTodoItemContents,
        deleteTeamTodoItemAll,
        changeFilter,
        addTeamMember } =  this.props;
    this.members = members;
    this.addTeamTodoItem = addTeamTodoItem;
    this.changeTeamTodoItemPriority = changeTeamTodoItemPriority;
    this.toggleTeamTodoItem = toggleTeamTodoItem;
    this.deleteTeamTodoItem = deleteTeamTodoItem;
    this.editTeamTodoItemContents = editTeamTodoItemContents;
    this.deleteTeamTodoItemAll = deleteTeamTodoItemAll;
    this.changeFilter = changeFilter;
    this.addTeamMember = addTeamMember;
```

취향 차이겠지만 이런 방식도 추천드립니다 😄

Reply:
오! 확실히 이전 코드보다 깔끔할 것 같네요 ㅎㅎ 감사합니당